modules/cluster-logging-rn-5.4.5.adoc

```diff
@@ -0,0 +1,22 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-5"]
+= Logging 5.5.1
+This release includes link:https://access.redhat.com/errata/RHSA-2022:6183[RHSA-2022:6183-OpenShift Logging Bug Fix Release 5.4.5].
+
+[id="openshift-logging-5-4-5-bug-fixes"]
+== Bug fixes
+* Before this update, the Operator did not ensure that the pod was ready, which caused the cluster to reach an inoperable state during a cluster restart. With this update, the Operator marks new pods as ready before continuing to a new pod during a restart, which resolves the issue. (link:https://issues.redhat.com/browse/LOG-2881[LOG-2881])
+
+* Before this update, the addition of multi-line error detection caused internal routing to change and forward records to the wrong destination. With this update, the internal routing is correct. (link:https://issues.redhat.com/browse/LOG-2946[LOG-2946])
+
```

Comment:
@libander we added 2 bugs into 5.4.5, do we need to add them here? https://issues.redhat.com/browse/LOG-3009 https://issues.redhat.com/browse/LOG-2972